AEPEdge.xcodeproj/project.pbxproj

```diff
@@ -1635,7 +1646,7 @@
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos appletvsimulator appletvos";
 				SWIFT_VERSION = 5.0;
 				TARGETED_DEVICE_FAMILY = "1,2,3";
-				TVOS_DEPLOYMENT_TARGET = 10.0;
+				TVOS_DEPLOYMENT_TARGET = 11.0;
 				WATCHOS_DEPLOYMENT_TARGET = 4.0;
```

Comment:
Is is possible to remove `MACOSX_DEPLOYMENT_TARGET` and `WATCHOS_DEPLOYMENT_TARGET` from this functional test configuration?